docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -46,8 +46,8 @@ import (
 	"github.com/spf13/pflag"
 	cliflag "k8s.io/component-base/cli/flag"
```

Comment:
@aniruddha2000 can you please drop this diff from the PR?

Reply:
It seems to be the same as was already merged in https://github.com/kubernetes-sigs/cluster-api/pull/7912/files

Reply:
Hm. Took a closer look. Not sure why the GitHub UI is showing the diff. I'll merge it should be fine.